hangups/conversation.py

```diff
@@ -11,34 +12,62 @@
 
 
 @asyncio.coroutine
-def build_user_conversation_list(client):
+def build_user_conversation_list(client, conversations_per_request=100):
```

Comment:
Yes, Google's clients lazy load them. Probably not strictly necessary to behave the same, agreed.